src/Files.App/Utils/Storage/Operations/FilesystemOperations.cs

```diff
@@ -547,15 +562,18 @@ public Task<IStorageHistory> RenameAsync(IStorageItem source, string newName, Na
 			return RenameAsync(StorageHelpers.FromStorageItem(source), newName, collision, progress, cancellationToken);
 		}
 
-		public async Task<IStorageHistory> RenameAsync(IStorageItemWithPath source,
-													   string newName,
-													   NameCollisionOption collision,
-													   IProgress<StatusCenterItemProgressModel> progress,
-													   CancellationToken cancellationToken,
-													   bool asAdmin = false)
+		public async Task<IStorageHistory> RenameAsync(
+			IStorageItemWithPath source,
+			string newName,
+			NameCollisionOption collision,
+			IProgress<StatusCenterItemProgressModel> progress,
+			CancellationToken cancellationToken,
+			bool asAdmin = false)
 		{
 			StatusCenterItemProgressModel fsProgress = new(progress, true, FileSystemStatusCode.InProgress);
 
+			// TODO: Get total size
```

Comment:
I don't think we need to the size for rename operations.

Reply:
Removed all